src/Tribe/Admin/Notice/Install_Event_Tickets.php

```diff
@@ -192,7 +193,67 @@ public function should_display_notice_activate(): bool {
 		 *
 		 * @param bool $should_display True if the notice should display.
 		 */
-		return apply_filters( 'tec_events_admin_notice_event_tickets_should_display', $this->is_installed() && ! $this->is_active() && ! $this->is_install_plugin_page() );
+		return apply_filters( 
+			'tec_events_admin_notice_event_tickets_should_display', 
+			$this->is_installed() && ! $this->is_active() && ! $this->is_install_plugin_page() && $this->is_tec_related_page()
+		);
+	}
+
+	/**
+	 * Checks if the current admin page is TEC related.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool True if the current admin page is TEC related.
+	 */
+	public function is_tec_related_page(): bool {
+		// Not in the admin we don't even care.
+		if ( ! is_admin() ) {
+			return false;
+		}
+
+		global $current_screen;
+		
+		// No screen, bail.
+		if ( empty( $current_screen ) ) {
+			return false;
+		}
+		
+		// Match any TEC post type screens (events, organizers, venues).
+		$tec_post_types = [
+			\Tribe__Events__Main::POSTTYPE,
+			\Tribe__Events__Organizer::POSTTYPE,
+			\Tribe__Events__Venue::POSTTYPE
+		];
+		
+		if ( in_array( $current_screen->post_type, $tec_post_types, true ) ) {
+			return true;
+		}
```

Comment:
For this kind of comparison, we can (and should) switch away from `in_array()`. This is how I would do it:

```suggestion
		$tec_post_types = [
			\Tribe__Events__Main::POSTTYPE      => true,
			\Tribe__Events__Organizer::POSTTYPE => true,
			\Tribe__Events__Venue::POSTTYPE     => true,
		];
		
		if ( isset( $tec_post_types[ $current_screen->post_type ] ) ) {
			return true;
		}
```

Reply:
Also, it would be nice to import the classes at the top of the file, and possibly also alias them at the same time.